test/e2e/specs/editor/block-computed-styles/paragraph-styles.spec.js

```diff
@@ -0,0 +1,402 @@
+						color: {
+							text: 'rgb(211, 211, 211)',
+						},
+					},
+				},
+			};
+
+			// The test asserts that block type styles take precedence over
+			// element styles so include them both in the global styles update.
+			await requestUtils.updateGlobalStyles( {
+				...globalElementStyles,
+				blocks: {
+					'core/paragraph': globalBlockStyles,
+				},
+			} );
+
+			await admin.createNewPost();
+			await editor.insertBlock( {
+				name: 'core/paragraph',
+				attributes: {
+					content: 'Hello <a href="https://wordpress.org">world</a>',
+				},
+			} );
+
+			const block = editor.canvas.getByRole( 'document', {
+				name: 'Block: Paragraph',
+			} );
+			await expect( block ).toHaveCSS(
+				'color',
+				globalBlockStyles.color.text
+			);
+			await expect( block ).toHaveCSS(
+				'background-color',
+				globalBlockStyles.color.background
+			);
+
+			const link = block.getByRole( 'link' );
+			await expect( link ).toHaveCSS(
+				'color',
+				globalBlockStyles.elements.link.color.text
+			);
+		} );
+
+		await test.step( 'Block instance styles', async () => {
+			const instanceStyles = {
+				color: {
+					text: 'rgb(12, 12, 12)',
+					background: 'rgb(112, 112, 112)',
+				},
+				elements: {
+					link: {
+						color: {
+							text: 'rgb(212, 212, 212)',
+						},
+					},
+				},
+			};
+
+			await admin.createNewPost();
+			await editor.insertBlock( {
+				name: 'core/paragraph',
+				attributes: {
+					content: 'Hello <a href="https://wordpress.org">world</a>',
+					style: instanceStyles,
+				},
+			} );
+
+			const block = editor.canvas.getByRole( 'document', {
+				name: 'Block: Paragraph',
+			} );
+			await expect( block ).toHaveCSS(
+				'color',
+				instanceStyles.color.text
+			);
+			await expect( block ).toHaveCSS(
+				'background-color',
+				instanceStyles.color.background
+			);
+			// Block gets automatic padding when a background is set on the instance.
+			await expect( block ).toHaveCSS( 'padding', '20px 38px' );
+
+			const link = block.getByRole( 'link' );
+			await expect( link ).toHaveCSS(
+				'color',
+				instanceStyles.elements.link.color.text
+			);
+		} );
+	} );
+
+	test( 'Typography', async ( { admin, editor, requestUtils } ) => {
+		const globalElementStyles = {
+			typography: {
+				fontWeight: '100',
+				fontSize: '11px',
+				textTransform: 'uppercase',
+				letterSpacing: '1.1px',
+				lineHeight: '1.1',
+			},
+		};
+
+		await test.step( 'Global element styles', async () => {
+			await requestUtils.updateGlobalStyles( {
+				...globalElementStyles,
+			} );
+
+			await admin.createNewPost();
+			await editor.insertBlock( {
+				name: 'core/paragraph',
+				attributes: {
+					content: 'Hello <a href="https://wordpress.org">world</a>',
+				},
+			} );
+
+			const block = editor.canvas.getByRole( 'document', {
+				name: 'Block: Paragraph',
+			} );
+
+			// fontStyle, fontWeight, fontSize.
+			await expect( block ).toHaveCSS(
+				'font',
+				'100 11px / 12.1px Times'
```

Comment:
It seems like the test is failing, probably because on linux it evaluates a different font name (Times vs Times New Roman).

I'll break this out into multiple assertions for each property to fix that.